---
layout: no-title-post
title: "About"
permalink: /about/
---

## __Education and Experience__

* __BSc in Biochemistry and Molecular Biology__ *([Universidad Rovira I Virgili](http://www.urv.cat/es/estudios/grados/oferta/graudebioquimica/), Spain)*<br>During these 4 years as an undergaduate, most of my practical sessions involved wet-lab experiments. Nevertheless, it was not until I took the bioinformatics course when I realised how much I enjoyed coding: if I properly told a computer to do <i>"something"</i> with my biological data, I actually got <i>"this something"</i> done (what I could not always say after my wet-lab practical sessions). Thanks to the Erasmus+ programme, I could work on a bioinformatics project at the [University of Manchester](http://www.manchester.ac.uk/), which was focused on troubleshooting and implementing a pipeline to carry out virtual screenings. From then onwards, I knew that my academic career had to integrate two fields: biology and informatics.
* __MSc in Bioinformatics__ *([Högskolan I Skövde](http://www.his.se/en/#), Sweden)*<br>After finishing the theoretical courses, I carried out my Master's dissertation at [Karolinska Institutet](http://ki.se/en/research/centre-for-translational-microbiome-research-ctmr), where I developed a pipeline to characterise bacterial isolates based on whole-genome sequencing data and other software pipelines to analyse transcriptomic and proteomic data. My dissertation ["BACTpipe: Characterization of bacterial isolates based on whole-genome sequence data" can be downloaded from free on the DIVA portal](http://his.diva-portal.org/smash/record.jsf?pid=diva2%3A1197064&dswid=-2253). `BACTpipe` has now been further developed and implemented with new features by the computational research team at the [CTMR](https://ki.se/en/research/centre-for-translational-microbiome-research-ctmr). The [open source code is available in the `BACTpipe` GitHub repository](https://github.com/ctmrbio/BACTpipe).
* __PhD in Computational Biology__ *([Queen Mary University of London](http://www.sbcs.qmul.ac.uk/), UK)*<br>Once I completed my MSc studies, I wanted to explore other fields in computational biology such as Bayesian statistics and modelling as I was keen on integrating a more mathematical background with my computational skills at that moment. To achieve this, I started my PhD at the [dos Reis Lab group](https://dosreislab.github.io/people/), at Queen Mary University of London, where I worked on the application of Bayesian MCMC statistical methods to study species divergences through time with both molecular and morphological quantitative data.
* __Post-doctoral researcher and Teaching fellow__ *([University College London](https://www.ucl.ac.uk/biosciences/departments/genetics-evolution-and-environment), UK)*<br>Currently, my main interest is focused on developing and applying new methods and pipelines to study evolutionary processes. I have been working on analyses for detecting positive selection and continuing to expand my research on Bayesian timetree inference with molecular and morphological datasets (with both extant and fossil species). My ultimate goal is to ensure that all the computational tools that I develop as part of the projects I am involved in are well documented, user-friendly, and reproducible.

<br>

## __Publications__

* [A Species-Level Timeline of Mammal Evolution Integrating Phylogenomic Data](https://rdcu.be/cDHW7)<br>
__Álvarez-Carretero S__<sup>*</sup>, Tamuri AU<sup>*</sup>, et al. (2021) Nature.<br><font size="2"><sup>*</sup>These authors contributed equally.</font><br> >> Resources available for this study:<br>
  - [Step-by-step tutorial to reproduce the results of this study](https://github.com/sabifo4/mammals_dating).
  - [Interview I had at the YouTube channel "Evolution soup" with regards to this research](https://www.youtube.com/watch?v=shsOpkkcq_A&feature=youtu.be). By watching this video, you can learn more about the most precise evolutionary timeline for mammals, each author's contribution, the outline and main outcome of this research, and a little bit more about my academic career.
  - [Press release in English](https://sruk.org.uk/study-offers-new-insights-into-the-timeline-of-mammal-evolution/).
  - [Press release in Spanish](https://sruk.org.uk/es/un-nuevo-estudio-proporciona-nuevas-respuestas-sobre-la-cronologia-evolutiva-de-los-mamiferos/).
<spacer></spacer>

* [Bayesian phylogenomic dating](https://link.springer.com/chapter/10.1007/978-3-030-60181-2_13)<br>
__Álvarez-Carretero S__ and dos Reis M. (2020) _In_: Ho S (ed.) The Molecular Evolutionary Clock: Theory and Practice (2021).
<spacer></spacer>

* [Dire wolves were the last of an ancient New World canid lineage](https://www.nature.com/articles/s41586-020-03082-x)<br>
Perri Angela R<sup>*</sup>, Mitchell Kieren J<sup>*</sup>, Mouton Alice<sup>*</sup>, __Álvarez-Carretero S__<sup>*</sup>, et al. (2021) Nature.<br><font size="2"><sup>*</sup>These authors contributed equally.</font><br>
<spacer></spacer>

* [Phylogenomic Resolution of the Cetacean Tree of Life Using Target Sequence Capture](https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz068/5601630)<br>
McGowen MR, Tsagkogeorga G, __Álvarez-Carretero S__, dos Reis M, Struebig M, Deaville R, Jepson PD, Jarman S, Polanowski A, Morin PA, Rossiter SJ. (2019) Systematic Biology.
<spacer></spacer>

* [Identification of Functional and Druggable Sites in Aspergillus fumigatus Essential Phosphatases by Virtual Screening](https://www.mdpi.com/1422-0067/20/18/4636/htm)<br>
Thornton BP, Johns A, Al-Shidhani R, __Álvarez-Carretero S__, Storer ISR, Bromley MJ, Tabernero L. (2019) Molecules.
<spacer></spacer>

* [Bayesian Estimation of Species Divergence Times Using Correlated Quantitative Characters](https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz015/5366706)<br>
__Álvarez-Carretero S__, Goswami A, Yang Z, dos Reis M. (2019) Systematic Biology.<br> >> Resources available for this study:<br>
  - [Step-by-step tutorial followed to carry out each computational task of this study](https://github.com/sabifo4/morpho).
  - The R functions described in this study were implemented in [the `mcmc3r` R package](https://github.com/dosreislab/mcmc3r), which includes [a vignette to reproduce the results obtained with the input data](https://github.com/dosreislab/mcmc3r/tree/master/vignettes).
  - Please install `mcmc3r` using the command `devtools::install_github("dosreislab/mcmc3r", build_vignettes = TRUE)` so you have access to the vignettes included in `mcmc3r` too.
<spacer></spacer>

* [The bile salt glycocholate induces global changes in gene and protein expression and activates virulence in enterotoxigenic Escherichia coli](http://www.mdpi.com/1420-3049/23/2/353)<br>
Joffre E, Nicklasson M, __Álvarez-Carretero S__, Xiao X, Sun L, Nookaew I, Zhu B, Sjöling Å. (2019) Scientific Reports.
<spacer></spacer>

* [VSpipe, an Integrated Resource for Virtual Screening and Hit Selection: Applications to Protein Tyrosine Phospahatase Inhibition](http://www.mdpi.com/1420-3049/23/2/353)<br>
__Álvarez-Carretero S__, Pavlopoulou N, Adams J, Gilsenan J, and Tabernero L. (2018) Molecules.<br> >> Resources available for this study:<br>
  - [`VSpipe` is an open-source code available for download on GitHub](https://github.com/sabifo4/VSpipe).
<spacer></spacer>

* [An evaluation of different partitioning strategies for Bayesian estimation of species divergence times](https://academic.oup.com/sysbio/article/doi/10.1093/sysbio/syx061/3921199/An-Evaluation-of-Different-Partitioning-Strategies)<br>
Konstantinos A, __Álvarez-Carretero S__, dos Reis M, and Yang Z. (2017) Systematic Biology.

<br>

## __Awards and Scholarships__

* [USyD-UCL Partnership Collaboration Awards (PCA)](https://www.sydney.edu.au/research/research-funding/research-fellowships.html)<br>
Application to fund the project “Reconstructing the evolutionary timescale of the Tree of Life” together with Prof Simon Ho. Awarded by the University of Sydney in 2022.
* [**Ernst Mayr Award**](https://www.systbio.org/ernst-mayr-award.html), awarded by the Society of Systematic Biologists.
* [**National awards after University degree completion during the 2014-15 academic course**](http://www.educacionyfp.gob.es/servicios-al-ciudadano/catalogo/general/99/998341/ficha/998341-2014-2015.html), awarded by Ministerio de Educación, Cultura y Deporte.<br>(Original award name: _Premios Nacionales de Fin de Carrera de Educación Universitaria para el curso académico 2014-2015_)   
* [**Student excellence award for the BSc in Biochemistry and Molecular Biology**](https://seuelectronica.urv.cat/media/upload/pdf//cg_20160225_punt71_premis_ext_grau_mast_corrsl_pend_sig_editora_4_1_2128.pdf), awarded by Universitat Rovira I Virgili.<br>(Original award name: _Premi Extraordinari de Final d’Estudis del grau de Bioquímica I Biologia Molecular_)
* [**Scholarship for research funding during the academic course 2014-15**](https://www.boe.es/boe/dias/2015/01/27/pdfs/BOE-A-2015-700.pdf), awarded by Ministerio de Educación, Cultura y Deporte in 2014.<br>(Original award name: _Becas de Colaboración 2014-15_)    

<br>

## __Teaching__

### Instructor
- <b>Bayesian statistics</b> [MSc CMEE, Imperial College] <i>(2021/22 & 2022/23).</i>
- <b>Phylogenomics course</b> [Physalia course] <i>(Dec 2021, [material and content available on GitHub](https://github.com/sabifo4/BayesianInference_practicals)</i>
- <b>Molecular Evolution</b> [BIOL0033, UCL] <i>(2020/21 – 2021/22).</i>
- <b>Research Computing Workshop – R, Bash, and Python</b> <i>(Feb & Jun 2020, workshop sponsored by [SRUK/CERU](https://sruk.org.uk), co-organised with [Dr Diego Alonso](https://www.imperial.ac.uk/people/d.alonso-alvarez). Material and content available on GitHub for [the R and Bash workshop](https://github.com/sabifo4/RnBash) and for the [Phyton workshop](https://github.com/dalonsoa/SRUK_Python_course)).</i>
- <b>A practical introduction to R and Python for researchers</b> <i>(Apr 2018 & Mar 2019, workshop sponsored by [SRUK/CERU](https://sruk.org.uk), co-organised with [Dr Diego Alonso](https://www.imperial.ac.uk/people/d.alonso-alvarez). [Material and content available on GitHub](https://github.com/sabifo4/II_SRUK_Scientific_Computing)).</i>
- <b>Morphometrics and the Procrustes alignment</b> <i>(2017-2021, workshop I organised to engage the NERC-DTP students with the research carried out at the [dos Reis lab](https://dosreislab.github.io/people/) during their induction at QMUL. [The material is available on my website as a post](https://sabifo4.github.io/blog/Morphometrics_and_Procrustes_alignment)).</i>

### Teaching Fellow
- <b>Biochemistry and Molecular Biology</b> [BIOC0001, UCL] <i>(2021/22).</i>
- <b>Basics for Molecular Biosciences</b> [BIOC0002, UCL] <i>(2021/22).</i>
- <b>Introduction to Microbiology</b> [BIOC0004, UCL] <i>(2021/22).</i>
- <b>Biochemistry and Molecular Biology</b> [BIOC0005, UCL] <i>(2021/22).</i>
- <b>Advanced Investigative projects in molecular biosciences</b> [BIOC0021, UCL] <i>(2021/22).</i>
- <b>MSci Biochemistry Year 4</b> [BIOC0024, BIOC0025, BIOC0026; UCL] <i>(2021/22).</i>

### Supervision
- <b>MSci project</b> <i>(2020/21) – Co-supervising Léa Cavalli’s research project with Prof Ziheng Yang | Dissertation: "A simulation study to explore the time-dependent rates effect in viral evolution".</i>

### Teaching Assistant
- <b>Computational Molecular Evolution course</b> <i>(May 2018, May 2019 & July 2022).</i>

### Demonstrator
- <b>Statistics for Bioinformaticians</b> [BIO782P, QMUL] <i>(2019/20).</i>
- <b>IT stats for Molecular and Cellular Biology</b> [BIO190, QMUL] <i>(2019/20).</i>
- <b>Practical laboratories and Seminars</b> [SEF033, QMUL] <i>(2018/19).
- <b>Stats & R</b> [BIO309, QMUL] <i>(2017/18 – 2019/20).</i>
- <b>Free style laboratories</b> [BIO193 & BIO199, QMUL] <i>(2016/17).</i>
- <b>Thinking workshops</b> [BIO193 & BIO199, QMUL] <i>(2016/17 – 2017/18).</i>
- <b>Dry lab tutorials</b> [BIO193 & BIO199, QMUL] <i>(2016/17).</i>
- <b>Biochemistry laboratories</b> [BIO199, QMUL] <i>(2016/17).</i>

<br>

## __Invited Talks and Science Communication <i>(outreach)</i>__

- <b>XXIII International Congress of Genetics</b> <i>(scheduled for July 2023) – Invited talk at the “Molecular Evolution” symposium at the Evolutionary Genetics session.</i>
- <b>Native Schools workshop</b> <i>(Nov 2022) – Scientific activity sponsored by Native Scientist aimed at promoting science and language among school pupils.</i>
- <b>Silwood Ecology and Evolution Seminar, Imperial College London</b> <i>(June 2022) – Invited talk (virtual).</i>
- <b>“Data, tools and applications: Creating a pipeline from Omics to AI” meeting, Alan Turing Institute</b> <i>(May 2022) – Invited talk.</i>
- <b>Brunel Bioinformatics Workshop 2022</b> <i>(May 2022) – Invited talk at the [“Statistical Modelling” session of the workshop](https://bbw2022.co.uk/speakers).</i>
- <b>Interview at “En Fase Experimental” radio podcast</b> <i>(May 2022) – Interview about the day-to-day of a computational biologist and my research in evolutionary biology (virtual). [You can listen to this podcast episode on the SRUK/CERU website](https://sruk.org.uk/es/efex64/)</i>
- <b>Seminar on Phylogenomics analyses with MCMCtree, University of Oklahoma</b> <i>(April 2022) – Invited talk (virtual).</i>
- <b>Interview by the students at IE Cañada Blanch for the International Day of Women and Girls in Science</b> <i>(Feb 2022) – Interview about my research and how it feels to pursue a career as a woman in STEM. You can see the videos with my answers to the students' questions [on YouTube](https://www.youtube.com/playlist?list=PLy_Y_fVXS6UxEmAPatrdVrLYzxAzDNNtP). This event was a collaboration between [IE Cañada Blanch](https://twitter.com/Canada_Blanch) and the [Office for Cultural and Scientific Affairs (Embassy of Spain in London)](https://twitter.com/SpainCultureUK).</i>
- <b>Interview at “Evolution Soup” YouTube channel</b> <i>(Jan 2022) – Interview about my research on mammals’ evolution and my academic career [available to watch on YouTube](https://www.youtube.com/watch?v=shsOpkkcq_A&feature=youtu.be) (virtual).</i>
- <b>Seminar at the Instituto de Ciencias Matemáticas, Consejo Superior de Investigaciones Científicas</b> <i>(Nov 2021) – Invited talk (virtual).</i>
- <b>Seminar at the Palaebiology Research group, University of Bristol</b> <i>(Jun 2021) – Invited talk (virtual).</i>
- <b>Graduate Phylogenomics seminar, Michigan State University</b> <i>(Feb 2021) – Invited talk (virtual).</i>
- <b>CLOE Joint Group Meeting</b>, University College London <i>(Nov 2020) – Invited talk (virtual).</i>
- <b>Back to present: Understanding evolution</b> <i>(Jun 2018 & Jun 2019) – Scientific activity for children presented during the V and VI SRUK/CERU International Symposia.</i>

<br>

## __Events Organisation and Participation__

- <b>XXIII International Congress of Genetics</b> <i>(Nov 2022 – Present) – Co-convenor at the “Molecular Evolution” symposium at the Evolutionary Genetics session.</i>
- <b>X SRUK/CERU International Symposium</b> <i>(Nov 2022 – Present) – Member of the organising committee (website development and management).</i>
- <b>CLOE Winter Quarterly Symposium</b> <i>(Oct – Dec 2021) – Member of the organising committee of the CLOE anniversary symposium. You can find [more information about this event and the invited speakers at the CLOE website](https://www.ucl.ac.uk/biosciences/gee/ucl-centre-lifes-origins-and-evolution)</i>
- <b>IX SRUK/CERU International Symposium</b> <i>(Nov 2021 – Jun 2022) – Member of the organising committee ([website development and management](https://sruk2022.com/)).</i>
- <b>SSB Awards Committee</b> <i>(June 2021) – Member of the judging panel for the talks presented at the SSB Excellence Symposium.</i>
- <b>VIII SRUK/CERU International Symposium (virtual)</b> <i>(Nov 2020 – Jun 2021) – Member of the organising committee ([website development and management](https://sruk2021.wordpress.com/)).</i>
- <b>VII SRUK/CERU International Symposium</b> <i>(Nov 2018 – Jun 2019) – Member of the organising committee ([website development and management](https://sruk2019.wordpress.com/)).</i>
- <b>VI SRUK/CERU International Symposium</b> <i>(Mar – May 2018) – Member of the organising committee ([website development and management](https://sruk2018symposium.wordpress.com/)).</i>
- <b>Phylogroup XI</b> <i>(Dec 2017 – Apr 2018) – Co-organiser of the meeting.</i>

<br>

## __Research Presentations__

- <b>Inferring species level timelines with phylogenomic data: A Bayesian sequential subtree approach</b> <i> (Jun and Aug 2022) – Virtual talk at the online session of the Evolution meeting and in-person talk at the ESEB conference.</i>
- <b>Estimating the divergence of dire wolves from their common ancestor with living canids using a Bayesian dating approach</b> <i> (Jun 2021) – Talk at the online Evolution meeting.</i>
- <b>Estimating the divergence of dire wolves from their common ancestor with living canids using a Bayesian dating approach</b> <i> (Nov 2019) – Poster presentation at the Young Systematists’ Forum meeting.</i>
- <b>Bayesian estimation of species divergence times using correlated quantitative characters</b> <i> (Jun 2019) – [Talk at the Evolution meeting awarded with the Ernst Mayr Award](https://www.systbio.org/ernst-mayr-award.html).</i>
- <b>Bayesian estimation of species divergence times using correlated quantitative characters</b> <i>(Nov 2018) – Talk at the Young Systematists’ Forum meeting.</i>
- <b>Comparison of relaxed-clock models using the Bayesian model selection implemented in MCMCtree</b> <i>(Aug 2018) – Poster presentation at the Evolution meeting, available at the [dos Reis lab](https://dosreislab.github.io/2018/08/22/evolution2018.html).</i>
- <b>Bayesian estimation of species divergence times using quantitative characters</b> <i>(Apr 2018) – Talk at Phylogroup XI meeting.</i> 

<br>

## __Society Memberships__

- <b>ESEB</b> <i>(May 2022 – Present)</i> – Regular member of the European Society of Evolutionary Biology.
- <b>SSE</b> <i>(Feb 2019 – Present)</i> – Regular member of the Society for the Study of Evolution.
- <b>SSB</b> <i>(Jan 2018 – Present)</i> – Regular member of the Society of Systematic Biologists.
- <b>SRUK/CERU</b> <i>(Nov 2017 - Present)</i> – Regular member of the [Society of Spanish Researchers in the United Kingdom (SRUK/CERU)](https://sruk.org.uk). Member of the SRUK/CERU London working group since 2017 and former secretary of the [SRUK/CERU London constituency](https://sruk.org.uk/constituencies/london/) (2019-2021). Member of the Press Department since 2018, [co-editor-in-chief of the SRUK/CERU blog](https://sruk.org.uk/teams/blog-committee/) since 2019, and current [director of the Press department](https://sruk.org.uk/teams/press-department-backstage/). [Website manager](https://sruk.org.uk/team/sandra-ac/) since 2021.

<br>

## __Peer-review__

Manuscripts reviewed for <i>Systematic Biology</i>, <i>Bioinformatics</i>, <i>Methods in Ecology and Evolution</i>, <i>Evolution</i>, <i>Cambridge Elements STM3</i>, <i>Nature Ecology and Evolution</i>, <i>Paleobiology</i>, <i>Proceedings of the Royal Society B</i>, <i>Nature Communications</i>, <i>Communications Biology</i>, <i>Bioinformatics Advances</i>, <i>Nature</i>, <i>Palaeontology</i>, <i>Genome Biology and Evolution</i>.
<space></space>
Grants reviewed for the 2019 and 2022 [Graduate Student Research Awards](https://www.systbio.org/graduate-student-research-awards.html) (SSB).

<br>

## __Additional Information__

Native speaker of Spanish, fluent in English and Catalan, basic level in Greek and Modern Standard Arabic. Mysore Ashtanga practitioner. Piano player and amateur composer.

<br>

---
<spacer></spacer>

Last update: 2023/01/19.
