---
layout: page
title: About
permalink: /about/
---

![image](/assets/figs/sac.jpg){: style="float: right" width="20%"}
Hi everyone!

I am Sandra Álvarez-Carretero, a computer scientist currently doing my PostDoc at [Ziheng Yang Lab](http://abacus.gene.ucl.ac.uk/) at University College London.



Even though now I cannot imagine a day working without my laptop, my career started
at the wet lab and, little by little, I specialised in computer science:

<br />
<br />

* __BSc in Biochemistry and Molecular Biology__ *([Universidad Rovira I Virgili](http://www.urv.cat/es/estudios/grados/oferta/graudebioquimica/), Spain)*
: During this 4-year BSc, most of the practicals were wet lab-related. However, it was not until I took the bioinformatics course that I realised how much 
I enjoyed writing a code to tell a computer to do __"something"__ with my 
biological data and actually getting __"this something"__ done 
(what I could not always say after my wet lab practicals...). Thanks to the Erasmus+ programme, I could work on a bioinformatics project at the [University of Manchester](http://www.manchester.ac.uk/),
which was focused on troubleshooting and implementing a pipeline to carry out virtual screenings.
From then onwards, I was sure that my next steps in science 
had to integrate two fields: biology and informatics.  

* __MSc in Bioinformatics__ *([Högskolan I Skövde](http://www.his.se/en/#), Sweden)*
: After finishing the theoretical courses, I carried out my Master's thesis at [Karolinska Institutet](http://ki.se/en/research/centre-for-translational-microbiome-research-ctmr), 
where I developed a pipeline to characterise bacterial isolates based on whole-genome sequencing data
 and other software pipelines to analyse transcriptomic and proteomic data. If you are interested in reading my dissertation, 
 "BACTpipe: Characterization of bacterial isolates based on whole-genome sequence data", it is available for download [here](http://his.diva-portal.org/smash/record.jsf?pid=diva2%3A1197064&dswid=-2253). `BACTpipe` has now been further implemented by the computational research team at the [CTMR](https://ki.se/en/research/centre-for-translational-microbiome-research-ctmr) and is available in [this GitHub repository](https://github.com/ctmrbio/BACTpipe).

* __PhD student__ *([Queen Mary University of London](http://www.sbcs.qmul.ac.uk/), UK)*
: Once I finished my MSc, I decided to explore other fields in computer science such as 
bayesian statistics and modelling where I could learn more about the maths used in this field. 
I carried out my PhD at [dos Reis Lab](dosreislab.github.io) research group, at Queen Mary University of Lonond,
working on the application of Bayesian MCMC statistical methods to 
study species divergences through time with both molecular and morphological quantitative data.

* __Post-doctoral researcher__ *([University College London](https://www.ucl.ac.uk/biosciences/departments/genetics-evolution-and-environment), UK)*
: Currently, my main interest is focused on developing and applying new methods to estimate species divergence times with morphological
quantitative data. I aim to use these new tools in timetree inference when combining large-scale morphological datasets (with both
extant and fossil species) with phylogenomic data sets. Assessing the mixing and computational limitations of MCMC algorithms in this type 
of analysis will be key to understand the compromise between computational time and data size.

<br />


# __Publications__

* [Bayesian phylogenomic dating](https://link.springer.com/chapter/10.1007/978-3-030-60181-2_13)   
   __Álvarez-Carretero S__ and dos Reis M. (2020) _In_: Ho S (ed.) The Molecular Evolutionary Clock: Theory and Practice (2021).

* [Dire wolves were the last of an ancient New World canid lineage](https://www.nature.com/articles/s41586-020-03082-x)   
   Perri Angela R<sup>*</sup>, Mitchell Kieren J<sup>*</sup>, Mouton Alice<sup>*</sup>, __Álvarez-Carretero S__<sup>*</sup>, et al. (2021) Nature.<br><font size="2"><sup>*</sup>These authors contributed equally.</font><br />

* [Phylogenomic Resolution of the Cetacean Tree of Life Using Target Sequence Capture](https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz068/5601630)   
   McGowen MR, Tsagkogeorga G, __Álvarez-Carretero S__, dos Reis M, Struebig M, Deaville R, Jepson PD, Jarman S, Polanowski A, Morin PA, Rossiter SJ. (2019) Systematic Biology.
   
* [Identification of Functional and Druggable Sites in Aspergillus fumigatus Essential Phosphatases by Virtual Screening](https://www.mdpi.com/1422-0067/20/18/4636/htm)   
   Thornton BP, Johns A, Al-Shidhani R, __Álvarez-Carretero S__, Storer ISR, Bromley MJ, Tabernero L. (2019) Molecules.   

* [Bayesian Estimation of Species Divergence Times Using Correlated Quantitative Characters](https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz015/5366706)  
   __Álvarez-Carretero S__, Goswami A, Yang Z, dos Reis M. (2019) Systematic Biology.   
   > The step-by-step tutorial followed to carry out each computational task of this study can be found [here](https://github.com/sabifo4/morpho).   
   > The R functions described in this study were implemented in the `mcmc3r` R package, available for download [here](https://github.com/dosreislab/mcmc3r). 

* [The bile salt glycocholate induces global changes in gene and protein expression and activates virulence in enterotoxigenic Escherichia coli](http://www.mdpi.com/1420-3049/23/2/353)  
   Joffre E, Nicklasson M, __Álvarez-Carretero S__, Xiao X, Sun L, Nookaew I, Zhu B, Sjöling Å. (2019) Scientific Reports.

* [VSpipe, an Integrated Resource for Virtual Screening and Hit Selection: Applications to Protein Tyrosine Phospahatase Inhibition](http://www.mdpi.com/1420-3049/23/2/353)  
   __Álvarez-Carretero S__, Pavlopoulou N, Adams J, Gilsenan J, and Tabernero L. (2018) Molecules.   
   > `VSpipe` is an open-source code available for download [here](https://github.com/sabifo4/VSpipe).

* [An evaluation of different partitioning strategies for Bayesian estimation of species divergence times](https://academic.oup.com/sysbio/article/doi/10.1093/sysbio/syx061/3921199/An-Evaluation-of-Different-Partitioning-Strategies)  
   Konstantinos A, __Álvarez-Carretero S__, dos Reis M, and Yang Z. (2017) Systematic Biology.

<br />

# __Outreach, events organisation, and research presentations__

#### *Academic course 2017-2018* 

   * **NERC DTP induction at QMUL** *(Nov 2017)*  
      Overview of the work carried out at dos Reis lab and about my PhD project. The practical part was focused on understanding the usage of the bioinformatics tools we use followed by a quick guide on a morphometrics analysis (acess [here](https://sabifo4.github.io/blog/Morphometrics_and_Procrustes_alignment)). 

   * **Phylogroup XI** *(Dec 2017 - Apr 2018)*  
      During these four months, I supported the organisation of the Phylogroup XI event. My main tasks were to collect the proposals for the oral presentations, to help decide which talks would be selected for the event, and organise the schedule of the event. 

   * **VI International Symposium SRUK/CERU** *(Mar 2018 - May 2018)* 
      During almost three months, I maintained and developed the website of the event as well as of the symposium forum for accommodation in Glasgow. 

   * **Bayesian estimation of species divergence times using quantitative characters** *(Apr 2018)*  
      Talk at Phylogroup XI about the preliminary results of my PhD research.
      
   * **Scientific Computing Workshop: A practical introduction to R and Python for researchers** *(Apr 2018)*   
      Event sponsored by the SRUK/CERU society that I co-organized with Dr. Diego Alonso.        

   * **Introduction to R** *(May 2018 | Computational Molecular Evolution course)*  
      Two-morning sessions about programming in R that I organized during the Computational Molecular Evolution course in Crete.  

   * **Back to present: Understanding evolution** *(Jun 2018)*  
      Activity that I prepared for the Science Zone as part of the VI International Symposium SRUK/CERU in Glasgow.  

   * **Comparison of relaxed-clock models using the Bayesian model selection implemented in MCMCtree (_Evolution 2018 meeting, poster_)** *(Aug 2018 | Evolution 2018 meeting, poster)*  
      Poster presentation at the joint Evolution 2018 meeting showing how to use the mcmc3r R package to estimate the marginal likelihood using the likelihood collected during MCMC chains ran in MCMCtree and compute Bayes factors for model selection. You can find the poster [here](https://dosreislab.github.io/2018/08/22/evolution2018.html).
      
#### *Academic course 2018/2019* 

   * **NERC DTP induction at QMUL** *(Nov 2018)*  
      Overview of the work carried out at dos Reis lab and a hands-on tutorial on how to generate a molecular alignment, estimate a molecular tree, and process morphometrics data.
      
   * **Bayesian estimation of species divergence times using correlated quantitative characters** *(Nov 2018 | YSF18 meeting, talk)*  
      Talk at the YSF2018 1-day symposium on the paper already submitted about the main method I had been working on my PhD.  
      
   * **VII International Symposium SRUK/CERU** *(Nov 2018 - Jun 2019)* 
      I was part of the organising committee of the symposium. My main task during these months was to develop and maintain the website to keep up with the latest updates regarding the event.   

   * **II Scientific Computing Workshop: A practical introduction to R and Python for researchers** *(Mar 2019)*   
      Event sponsored by the SRUK/CERU society that I co-organized with Dr. Diego Alonso.   
     
   * **Introduction to R** *(May 2019 | Computational Molecular Evolution course)*  
      One-morning session about programming in R that I organized during the Computational Molecular Evolution course in Hinxton.
      
   * **Back to present: Understanding evolution** *(Jun 2018)*  
      Activity that I prepared for the Science Zone as part of the VII International Symposium SRUK/CERU in Liverpool. 

   * **Bayesian estimation of species divergence times using correlated quantitative characters** *(June 2019 | Evolution 2019 meeting, awards talk)*  
      Talk at the Evolution 2019 meeting, with which I was granted the [**Ernst Mayr award**](https://www.systbio.org/ernst-mayr-award.html).
      
#### *Academic course 2019/2020* 

   * **NERC DTP induction at QMUL** *(Nov 2019)*  
      Overview of the work carried out at dos Reis lab and a hands-on tutorial on how to generate a molecular alignment, estimate a molecular tree, and process morphometrics data.   
      
   * **Estimating the divergence of dire wolves from their common ancestor with living canids using a Bayesian dating approach** *(Nov 2019 | YSF19 meeting, poster)*   
      Poster presentation at the YSF2019 1-day symposium on the submitted manuscript about the evolution of canids with a special focus on the dispersal and speciation of dire wolves.   

#### *Academic course 2020/2021* 

   * **NERC DTP induction at QMUL** *(Nov 2020)*  
      Overview of the work carried out at dos Reis lab and a hands-on tutorial on how to generate a molecular alignment, estimate a molecular tree, and process morphometrics data.   
      
   * **Bayesian estimation of species divergence times using GMM data** *(Nov 2020)*   
      Talk at the CLOE seminar organised by UCL.   

<br />

# __Teaching__

#### *Academic course 2016 - 2017*  
   
   * **Demonstrator (TA): Biochemistry laboratories**  
   Module BIO199, Undergraduate level (year 1), Queen Mary University of London.
   
   * **Demonstrator (TA): Dry lab tutorials**  
   Modules BIO193 and BIO199, Undergraduate level (year 1), Queen Mary University of London.
   
   * **Demonstrator (TA): Thinking workshops**  
   Modules BIO193 and BIO199, Undergraduate level (year 1), Queen Mary University of London.
   
   * **Demonstrator (TA): Free style laboratories**  
   Modules BIO193 and BIO199, Undergraduate level (year 1), Queen Mary University of London.

#### *Academic course 2017 - 2018*
   
   * **Demonstrator (TA): Stats & R**  
   Module BIO309, Undergraduate level (year 3), Queen Mary University of London.
   
   * **Demonstrator (TA): Thinking workshops**  
   Modules BIO193 and BIO199, Undergraduate level (year 1), Queen Mary University of London.   
   
   * **Teaching Assistant (TA): Computational Molecular Evolution course**   
   Providing support to the participants which ranged from software installation and usage to the theoretical background of using these bioinformatics tools to study molecular evolution.
     
#### *Academic course 2018 - 2019*
   
   * **Demonstrator (TA): Stats & R**  
   Module BIO309, Undergraduate level (year 3), Queen Mary University of London.
   
   * **Demonstrator (TA): Practical laboratories and Seminars**  
   Modules SEF033, Foundation level, Queen Mary University of London.   
   
   * **Teaching Assistant (TA): Computational Molecular Evolution course**   
   Providing support to the participants which ranged from software installation and usage to the theoretical background of using these bioinformatics tools to study molecular evolution.
 
#### *Academic course 2019 - 2020*
   
   * **Demonstrator (TA): Stats & R**  
   Module BIO309, Undergraduate level (year 3), Queen Mary University of London.
   
   * **Demonstrator (TA): IT stats for Molecular and Cellular Biology practicals**  
   Module BIO190, Undergraduate level (year 1), Queen Mary University of London.
   
   * **Demonstrator (TA): Statistics for Bioinformaticians**  
   Module BIO782P, Graduate level (MSc programme in Bioinformatics), Queen Mary University of London.   
   
#### *Academic course 2020 - 2021*
   
   * **Instructor: Molecular Evolution**  
   Module BIOL0033, MSci programme (year 4), University College of London.
    
   
   
<br />

# __Awards and scholarships__

   * [**Scholarship for research funding during the academic course 2014-15**](https://www.boe.es/boe/dias/2015/01/27/pdfs/BOE-A-2015-700.pdf), awarded by Ministerio de Educación, Cultura y Deporte in 2014.   
   (Original award name: _Becas de Colaboración 2014-15_)    
   * [**Student excellence award for the BSc in Biochemistry and Molecular Biology**](https://seuelectronica.urv.cat/media/upload/pdf//cg_20160225_punt71_premis_ext_grau_mast_corrsl_pend_sig_editora_4_1_2128.pdf), awarded by Universitat Rovira I Virgili.   
   (Original award name: _Premi Extraordinari de Final d’Estudis del grau de Bioquímica I Biologia Molecular_)   
   * [**National awards after University degree completion during the 2014-15 academic course**](http://www.educacionyfp.gob.es/servicios-al-ciudadano/catalogo/general/99/998341/ficha/998341-2014-2015.html), awarded by Ministerio de Educación, Cultura y Deporte.   
   (Original award name: _Premios Nacionales de Fin de Carrera de Educación Universitaria para el curso académico 2014-2015_)   
   * [**Ernst Mayr Award**](https://www.systbio.org/ernst-mayr-award.html), awarded by the Society of Systematic Biologists.

<br />

# __Society Memberships__

   * **SRUK/CERU** - Society of Spanish Researchers in the UK  *(Nov 2017 - Present)*   
      Member of the SRUK/CERU London working group since 2017 and [**secretary of the SRUK/CERU London constituency**](https://sruk.org.uk/constituencies/london/) since 2019. [**London press officer and SRUK/CERU blog editor**](https://sruk.org.uk/about-us/the-team/press-department/) since 2018 in the press department and [**co-editor-in-chief of the SRUK/CERU blog**](https://sruk.org.uk/blog/) since 2019 in the blog department.
   * **SSB** – Society of Systematic Biologists *(Jan 2018 - Present)*
   * **SSE** – Society for the Study of Evolution *(Feb 2019 - Present)* 

<br />

# __Peer-review__

Reviewer for _Systematic Biology_, _Bioinformatics_, _Methods in Ecology and Evolution_, _Evolution_, _Cambridge Elements STM3_, and _Nature Ecology and Evolution_.

----

Last update: 2021/01/20.
