---
layout: no-title-post
title: "About"
permalink: /about/
---


## __Education and Experience__

### Current

* __Research Fellow__ *([University College London](https://www.ucl.ac.uk/biosciences/departments/genetics-evolution-and-environment), UK)* & __Honorary Research Fellow__ *([University of Bristol](https://www.bristol.ac.uk/earthsciences/), UK)*<br>My main interest is focused on developing and applying reproducible methods to study evolutionary processes. Over the years, I have specialised on Bayesian timetree inference analyses with both molecular and morphological continuous data (with both extant and fossil species). As part of the project I have joined at University College London, I am currently working on timetree inference, focusing on MCMC algorithms and the assembly of empirical datasets to study human speciation and diversification (aDNA), the timescale of vertebrate terrestrialisation, and reconstructing the eukaryotic Tree of Life.<br><br> My ultimate goal is to ensure that all the computational tools that I develop as part of the projects I am involved in are well documented, user-friendly, and reproducible.

### Past

* __Senior Research Associate in Phylogenomics__ *([University of Bristol](https://www.bristol.ac.uk/earthsciences/), UK)*<br>As part of the [Biological Purpose project](https://www.biologicalpurpose.org/subaward-project/chance-versus-purpose-biosphere-evolution) I joined while in Bristol, I developed and applied new methods for deep timetree inference with the aim to unravel the patterns of early evolution. [We applied of such methods, cross-bracing, to infer the age of the Last Universal Common Ancestor](https://www.nature.com/articles/s41559-024-02461-1).
* __Postdoctoral researcher__ *([University College London](https://www.ucl.ac.uk/biosciences/departments/genetics-evolution-and-environment), UK)*<br>During these two-year position, I finished my [project on mammal evolution using the sequential-subtree approach](https://rdcu.be/cDHW7), worked on analyses for detecting positive selection, collaborated with various media and institutions for scientific outreach, and joined various projects focused on phylogenomic clock-dating analyses. During the academic year 2021-22, I was also a <b>teaching fellow</b> at UCL as part of a secondment, period during which I had the opportunity to further develop my teaching skills.
* __PhD in Computational Biology__ *([Queen Mary University of London](http://www.sbcs.qmul.ac.uk/), UK)*<br>Once I completed my MSc studies, I wanted to explore other fields in computational biology such as Bayesian statistics and modelling as I was keen on integrating a more mathematical background with my computational skills at that moment. To achieve this, I started my PhD at the [dos Reis Lab](https://dosreislab.github.io/people/), at Queen Mary University of London, where I worked on the application of Bayesian MCMC statistical methods to study species divergences through time with both molecular and morphological quantitative data.
* __MSc in Bioinformatics__ *([Högskolan I Skövde](http://www.his.se/en/#), Sweden)*<br>After finishing the theoretical courses, I carried out my Master's dissertation at [Karolinska Institutet](http://ki.se/en/research/centre-for-translational-microbiome-research-ctmr), where I developed a pipeline to characterise bacterial isolates based on whole-genome sequencing data and other software pipelines to analyse transcriptomic and proteomic data. My dissertation ["BACTpipe: Characterization of bacterial isolates based on whole-genome sequence data" can be accessed on the DIVA portal (free download)](http://his.diva-portal.org/smash/record.jsf?pid=diva2%3A1197064&dswid=-2253). Since I finished my MSc research project, the computational research team at the [CTMR](https://ki.se/en/research/centre-for-translational-microbiome-research-ctmr) has been implementing new features in `BACTpipe`. The [current open source code is available as part of the `BACTpipe` GitHub repository](https://github.com/ctmrbio/BACTpipe).
* __BSc in Biochemistry and Molecular Biology__ *([University Rovira I Virgili](http://www.urv.cat/es/estudios/grados/oferta/graudebioquimica/), Spain)*<br>During these 4 years as an undergraduate, most of my practical sessions involved wet-lab experiments. Nevertheless, it was not until I took the bioinformatics course when I realised how much I enjoyed coding: if I properly told a computer to do <i>"something"</i> with my biological data, I actually got <i>"this something"</i> done (what I could not always say after my wet-lab practical sessions). Thanks to the Erasmus+ programme, I could work on a bioinformatics project at the [University of Manchester](http://www.manchester.ac.uk/), which was focused on troubleshooting and implementing a pipeline to carry out virtual screenings. From then onwards, I knew that my academic career had to integrate two fields: biology and informatics.

<br>

## __Publications__

* [__A geological timescale for bacterial evolution and oxygen adaptation__](https://doi.org/10.1073/pnas.2406344122) <br>
Davín AA, Woodcroft BJ, Soo RM, Morel B, Murali R, Schrempf D, Clark JW, __Álvarez-Carretero S__, Boussau B, Moody ERR, Szánthó LL, Richy E, Pisani D, Hemp J, Fischer WW, Donoghue PCJ, Spang A, Hugenholtz P, Williams TA, Szöllősi GJ. (2025) Science.<br> >> Resources available for this study:<br>
  * [Tutorial benchmarking `MCMCtree` vs `McmcDate` when using the two different approximate likelihood calculation approaches impelemented in these two software](https://github.com/dschrempf/mcmc-date/tree/master/bench/comparison_with_mcmctree).
  * [Supplementary Material](https://www.science.org/doi/suppl/10.1126/science.adp1853/suppl_file/science.adp1853_sm.pdf).

<spacer></spacer>

* [__Persistence selection between simulated biogeochemical cycle variants for their distinct effects on the Earth system__](https://doi.org/10.1073/pnas.2406344122) <br>
Boyle RA, Moody ERR, Babcock G, McShea DN, __Álvarez-Carretero S__, Lenton TM, Donoghue PCJ. (2025) Proc Natl Acad Sci U S A.<br> >> Resources available for this study:<br>
  * [GitHub repository with data, code, and step-by-step guidelines for reproducing the timetree inference results described in our study](https://github.com/sabifo4/anoxphoto-divtimes).
  * [Supplementary Material](https://www.pnas.org/doi/suppl/10.1073/pnas.2406344122/suppl_file/pnas.2406344122.sapp.pdf).

<spacer></spacer>

* [__A timescale for the evolutionary history of sea spiders (Arthropoda: Pycnogonida)__](https://doi.org/10.1093/evolinnean/kzaf001) <br>
Nava M, __Álvarez-Carretero S__, Garwood RJ, Donoghue PCJ, Sabroux R, Pisani D. (2025) Evol J Linn Soc.<br> >> Resources available for this study:<br>
  *  [GitHub repository with data, code, and step-by-step guidelines for reproducing the timetree inference results described in our study](https://github.com/sabifo4/cspiders-divtimes).
  *  [All data required to interpret, verify, and reproduce the results regarding the timetree inference analyses and the rest of analyses that were carried out as part of this project can be found at our FigShare repository](https://figshare.com/s/f41a2898b504bd080e9c).
  *  [Supplementary Material](https://academic.oup.com/evolinnean/article/4/1/kzaf001/7979314#supplementary-data).

<spacer></spacer>

* [__The nature of the last universal common ancestor and its impact on the early Earth system__](https://doi.org/10.1038/s41559-024-02461-1)<br>
Moody ERR., __Álvarez-Carretero S__, Mahendrarajah TA, et al. (2024) Nat Ecol Evol.<br> >> Resources available for this study:<br>
  * [GitHub repository with data, code, and step-by-step guidelines for reproducing the timetree inference results described in our study](https://github.com/sabifo4/LUCA-divtimes). We also have the [GitHub repository we compressed in June 2024 with all the files we have used and generated as part of this project, which is available at the University of Bristol data repository](https://data.bris.ac.uk/data/dataset/405xnm7ei36d2cj65nrirg3ip). If you use these analyses, please cite the respository accordingly alongside the main article.
  * [All data required to interpret, verify, and extend the results regarding the reconciliation and phylogenomic analyses in this study can be found at our FigShare repository](https://doi.org/10.6084/m9.figshare.24428659).
  * [Additional data are available at the University of Bristol data repository](https://data.bris.ac.uk/data/dataset/405xnm7ei36d2cj65nrirg3ip).
  * [Article Moody ERR, who led the project, and I wrote for The Conversation](https://theconversation.com/how-we-reconstructed-the-ancestor-of-all-life-on-earth-236452).
  * [Supplementary Material](https://static-content.springer.com/esm/art%3A10.1038%2Fs41559-024-02461-1/MediaObjects/41559_2024_2461_MOESM1_ESM.pdf).

<spacer></spacer>

* [__VSpipe-GUI, an Interactive Graphical User Interface for Virtual Screening and Hit Selection__](https://doi.org/10.3390/ijms25042002)<br>
Hussain R, Hackett AS, __Álvarez-Carretero S__, Tabernero L. (2024) Int J Mol Sci.<br> >> Resources available for this study:<br>
  * [Supplementary Material](https://www.mdpi.com/article/10.3390/ijms25042002/s1).
  
<spacer></spacer>

* [__Beginner’s guide on the use of PAML to detect positive selection__](https://doi.org/10.1093/molbev/msad041)<br>
__Álvarez-Carretero S__<sup>*</sup>, Kapli P<sup>*</sup>, Yang Z (2023) Mol Biol Evol.<br><font size="2"><sup>*</sup>These authors contributed equally.</font><br> >> Resources available for this study:<br>
  * [GitHub repository with data, code, and step-by-step guidelines for reproducing the results of the analyses described in the `CODEML` protocol](https://github.com/abacus-gene/paml-tutorial/tree/main/positive-selection).
  * [Supplementary material where we discuss (i) analyses and checks you should carry out before running tests of positive selection with `CODEML`, (ii) gene tree VS species tree, and (iii) the usage of rooted and unrooted trees](http://abacus.gene.ucl.ac.uk/ziheng/pdf/2023Alvarez-Carretero-codeml-SI.pdf).
  
<spacer></spacer>

* [__A species-level timeline of mammal evolution integrating phylogenomic data__](https://rdcu.be/cDHW7)<br>
__Álvarez-Carretero S__<sup>*</sup>, Tamuri AU<sup>*</sup>, et al. (2022) Nature.<br><font size="2"><sup>*</sup>These authors contributed equally.</font><br> >> Resources available for this study:<br>
  * [Step-by-step tutorial to reproduce the results of this study](https://github.com/sabifo4/mammals_dating).
  * [Interview I had at the YouTube channel "Evolution soup" with regards to this research](https://www.youtube.com/watch?v=shsOpkkcq_A&feature=youtu.be). By watching this video, you can learn more about the most precise estimated evolutionary timeline for mammals, each author's contribution, the outline and main outcome of this research, and a little bit more about my academic career.
  * [Press release in English](https://sruk.org.uk/study-offers-new-insights-into-the-timeline-of-mammal-evolution/).
  * [Press release in Spanish](https://sruk.org.uk/es/un-nuevo-estudio-proporciona-nuevas-respuestas-sobre-la-cronologia-evolutiva-de-los-mamiferos/).
  * [Supplementary Material](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-021-04341-1/MediaObjects/41586_2021_4341_MOESM1_ESM.pdf).
  
<spacer></spacer>

* [__Bayesian phylogenomic dating__](https://link.springer.com/chapter/10.1007/978-3-030-60181-2_13)<br>
__Álvarez-Carretero S__ and dos Reis M. (2021) *In*: Ho S (ed.) The Molecular Evolutionary Clock: Theory and Practice (2021).
<spacer></spacer>

* [__Dire wolves were the last of an ancient New World canid lineage__](https://www.nature.com/articles/s41586-020-03082-x)<br>
Perri Angela R<sup>*</sup>, Mitchell Kieren J<sup>*</sup>, Mouton Alice<sup>*</sup>, __Álvarez-Carretero S__<sup>*</sup>, et al. (2021) Nature.<br><font size="2"><sup>*</sup>These authors contributed equally.</font><br> >> Resources available for this study:<br>
  * [Supplementary Material](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-020-03082-x/MediaObjects/41586_2020_3082_MOESM1_ESM.pdf).

<spacer></spacer>

* [__Phylogenomic resolution of the Cetacean Tree of Life using target sequence capture__](https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz068/5601630)<br>
McGowen MR, Tsagkogeorga G, __Álvarez-Carretero S__, dos Reis M, Struebig M, Deaville R, Jepson PD, Jarman S, Polanowski A, Morin PA, Rossiter SJ. (2020) Systematic Biology.<br> >> Resources available for this study:<br>
  * [Supplementary Material](https://doi.org/10.5061/dryad.jq40b0f).

<spacer></spacer>

* [__Identification of functional and druggable sites in _Aspergillus fumigatus_ essential phosphatases by virtual screening__](https://doi.org/10.3390/ijms20184636)<br>
Thornton BP, Johns A, Al-Shidhani R, __Álvarez-Carretero S__, Storer ISR, Bromley MJ, Tabernero L. (2019) Molecules.<br> >> Resources available for this study:<br>
  * [Supplementary Material](https://www.mdpi.com/1422-0067/20/18/4636/s1).

<spacer></spacer>

* [__Bayesian estimation of species divergence times using correlated quantitative characters__](https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz015/5366706)<br>
__Álvarez-Carretero S__, Goswami A, Yang Z, dos Reis M. (2019) Systematic Biology.<br> >> Resources available for this study:<br>
  * [Step-by-step tutorial followed to carry out each computational task of this study](https://github.com/sabifo4/morpho).
  * The R functions described in this study were implemented in [the `mcmc3r` R package](https://github.com/dosreislab/mcmc3r), which includes [a vignette to reproduce the results obtained with the input data](https://github.com/dosreislab/mcmc3r/tree/master/vignettes).
  * Please install `mcmc3r` using the command `devtools::install_github("dosreislab/mcmc3r", build_vignettes = TRUE)` so you have access to the vignettes included in `mcmc3r` too.
  * [Supplementary Material](http://doi.org/10.5061/dryad.q7rf263).

<spacer></spacer>

* [__The bile salt glycocholate induces global changes in gene and protein expression and activates virulence in enterotoxigenic Escherichia coli__](https://www.nature.com/articles/s41598-018-36414-z)<br>
Joffre E, Nicklasson M, __Álvarez-Carretero S__, Xiao X, Sun L, Nookaew I, Zhu B, Sjöling Å. (2019) Scientific Reports.<br> >> Resources available for this study:<br>
  * [Supplementary Material](https://static-content.springer.com/esm/art%3A10.1038%2Fs41598-018-36414-z/MediaObjects/41598_2018_36414_MOESM1_ESM.pdf).

<spacer></spacer>

* [__VSpipe, an integrated resource for virtual screening and hit selection: applications to protein tyrosine phospahatase inhibition__](http://www.mdpi.com/1420-3049/23/2/353)<br>
__Álvarez-Carretero S__, Pavlopoulou N, Adams J, Gilsenan J, Tabernero L. (2018) Molecules.<br> >> Resources available for this study:<br>
  * [`VSpipe` is an open-source code available for download on GitHub](https://github.com/sabifo4/VSpipe).

<spacer></spacer>

* [__An evaluation of different partitioning strategies for Bayesian estimation of species divergence times__](https://doi.org/10.1093/sysbio/syx061)<br>
Konstantinos A, __Álvarez-Carretero S__, dos Reis M, Yang Z. (2018) Systematic Biology.<br> >> Resources available for this study:<br>
  * [Supplementary Material](http://dx.doi.org/10.5061/dryad.d7839).

<br>

## __Awards and Scholarships__

* [__ESEB Conference Travel Award__](https://eseb.org/prizes-funding/conference-travel-award/)<br>
Travel Award that was used to cover travel expenses to attend the Evolution meeting that took place in Montréal in July 2024.
* [__SRUK/CERU Professional Development Award__](https://sruk.org.uk/initiatives/funding-grants-awards/travel-grants/)<br>
Travel Award that was used to cover travel expenses to attend the XXIII International Congress of Genetics that took place in Melbourne in July 2023.
* [__USyD-UCL Partnership Collaboration Awards (PCA)__](https://www.sydney.edu.au/research/research-funding/research-fellowships.html)<br>
Application to fund the project “Reconstructing the evolutionary timescale of the Tree of Life” together with Prof Simon Ho. Awarded by the University of Sydney in 2022.
* [__Ernst Mayr Award__](https://www.systbio.org/ernst-mayr-award.html), awarded by the Society of Systematic Biologists.
* [__National awards after University degree completion during the 2014-15 academic course__](http://www.educacionyfp.gob.es/servicios-al-ciudadano/catalogo/general/99/998341/ficha/998341-2014-2015.html), awarded by Ministerio de Educación, Cultura y Deporte.<br>(Original award name: *Premios Nacionales de Fin de Carrera de Educación Universitaria para el curso académico 2014-2015*).
* [__Student excellence award for the BSc in Biochemistry and Molecular Biology__](https://seuelectronica.urv.cat/media/upload/pdf//cg_20160225_punt71_premis_ext_grau_mast_corrsl_pend_sig_editora_4_1_2128.pdf), awarded by University Rovira I Virgili.<br>(Original award name: *Premi Extraordinari de Final d’Estudis del grau de Bioquímica I Biologia Molecular*).
* [__Scholarship for research funding during the academic course 2014-15__](https://www.boe.es/boe/dias/2015/01/27/pdfs/BOE-A-2015-700.pdf), awarded by Ministerio de Educación, Cultura y Deporte in 2014.<br>(Original award name: *Becas de Colaboración 2014-15*).

<br>

## __Teaching__

### Instructor

* <b>[Bayesian Phylogenetics Workshop](https://systass.org/workshop-bayesian-phylogenetics/)</b> [Sponsors: Society of Spanish Researchers in the United Kingdom (SRUK/CERU) the Systematics Association] <i>(Apr 2025). [Resources that I created for the Bayesian timetree inference session I taught are available on the `PAML` GitHub repository](https://github.com/abacus-gene/paml-tutorial/tree/main/mcmctree-approxlnL-aa).</i>
* <b>[Workshop on Computational Genomics](https://sishuowang2022.wordpress.com/2025/02/23/ad-computational-genomics-workshop-in-science-park-hong-kong-sar/)</b> [Sun Yat-sen University Institute of Advanced Studies Hong Kong] <i>(Mar 2025). [Resources that I created for the Bayesian timetree inference session I taught are available on the `PAML` GitHub repository](https://github.com/abacus-gene/paml-tutorial/tree/main/mcmctree-approxlnL-aa).</i>
* <b>[Phylogenomics and Population Genomics Advanced Course](https://www.ub.edu/certfem/ppgcourse/schedule)</b> [University of Barcelona] <i>(Jun 2023 & 2024). [Resources that I created for the Bayesian timetree inference session I taught are available on the PPG course GitHub repository](https://github.com/ppgcourseUB/ppgcourse2024_week1/tree/main/BayesianTimetreInference-SandraAlvarezCarretero).</i>
* <b>[ITAPHY - Phylogenetics workshop](https://sites.google.com/view/itaphylogeneticsworkshop)</b> [University of Trento] <i>(Jul 2023). [Resources that I created for the genome-scale dating session have been moved to my GitHub repository `PhyloPhysalia24`](https://github.com/sabifo4/PhyloPhysalia24).</i>
* <b>Computational Molecular Evolution course</b> <i>(May 2023, 2024 & 2025). [Information about the latest workshop in 2025](https://meetings.embo.org/event/25-comp-mol-evolution). [Resources that I created for the Bayesian timetree inference session I taught are available on the `PAML` GitHub repository](https://github.com/abacus-gene/paml-tutorial/tree/main/mcmctree-approxlnL-aa).</i></i>
* <b>Evolutionary Analysis Using `PAML` workshop</b> [USyD-UCL PCA] <i>(Jan 2023).</i>
* <b>Bayesian statistics</b> [MSc CMEE, Imperial College] <i>(2021/22 & 2022/23).</i>
* <b>Phylogenomics course</b> [Physalia course] <i>(Dec 2021, 2023 & 2024). [Information about the next course in 2025](https://www.physalia-courses.org/courses-workshops/phylogenomics/). [Resources that I created for the practical sessions I taught are available on my GitHub repository `PhyloPhysalia24`](https://github.com/sabifo4/PhyloPhysalia24).</i>
* <b>Molecular Evolution</b> [BIOL0033, UCL] <i>(2020/21, 2021/22 & 2024/25).</i>
* <b>Research Computing Workshop – R, Bash, and Python</b> <i>(Feb & Jun 2020). Workshop sponsored by [SRUK/CERU](https://sruk.org.uk), co-organised with [Dr Diego Alonso](https://www.imperial.ac.uk/people/d.alonso-alvarez). Material and content available on GitHub for [the R and Bash workshop](https://github.com/sabifo4/RnBash) and for the [Phyton workshop](https://github.com/dalonsoa/SRUK_Python_course).</i>
* <b>A practical introduction to R and Python for researchers</b> <i>(Apr 2018 & Mar 2019). Workshop sponsored by [SRUK/CERU](https://sruk.org.uk), co-organised with [Dr Diego Alonso](https://www.imperial.ac.uk/people/d.alonso-alvarez). [Material and content available on GitHub](https://github.com/sabifo4/II_SRUK_Scientific_Computing).</i>
* <b>Morphometrics and the Procrustes alignment</b> <i>(2017-2021). Workshop I organised to engage the NERC-DTP students with the research carried out at the [dos Reis lab](https://dosreislab.github.io/people/) during their induction at QMUL. [The material is available on my website as a post](https://sabifo4.github.io/blog/Morphometrics_and_Procrustes_alignment).</i>

### Teaching Fellow

* <b>Biochemistry and Molecular Biology</b> [BIOC0001, UCL] <i>(2021/22).</i>
* <b>Basics for Molecular Biosciences</b> [BIOC0002, UCL] <i>(2021/22).</i>
* <b>Introduction to Microbiology</b> [BIOC0004, UCL] <i>(2021/22).</i>
* <b>Biochemistry and Molecular Biology</b> [BIOC0005, UCL] <i>(2021/22).</i>
* <b>Advanced Investigative projects in molecular biosciences</b> [BIOC0021, UCL] <i>(2021/22).</i>
* <b>MSci Biochemistry Year 4</b> [BIOC0024, BIOC0025, BIOC0026; UCL] <i>(2021/22).</i>

### Supervision and Research Guidance

* <b>PhD project</b> <i>(2023-TBC) – Co-supervising Sidney Davies with Prof Phil Donoghue | Thesis: TBC.</i>
* <b>MSc project</b> <i>(2023) – Morena Nava’s research project with Prof Phil Donoghue and Prof
Davide Pisani | Dissertation: "A timescale to Sea Spider’s Evolution (Arthropoda: Pycnogonida)".</i>
* <b>MSci project</b> <i>(2020/21) – Co-supervising Léa Cavalli’s research project with Prof Ziheng Yang | Dissertation: "A simulation study to explore the time-dependent rates effect in viral evolution".</i>
* **Additional supervision and/or research guidance**: Since 2017, I have been providing guidance to students and members of staff on a request basis about how to run and/or improve their computational analyses, specially troubleshooting and fixing code as well as providing them with help to run phylogenetic software (e.g., `raxml-ng`, `MCMCtree`, `CODEML`, `BPP`, etc.). Most of this research guidance has led me to develop new protocols that users can easily customise to fit their datasets while, at the same time, they can run various analysis in a reproducible manner.

### Teaching Assistant

* <b>Computational Molecular Evolution course</b> <i>(May 2018, May 2019 & Jul 2022).</i>

### Demonstrator

* <b>Statistics for Bioinformaticians</b> [BIO782P, QMUL] <i>(2019/20).</i>
* <b>IT stats for Molecular and Cellular Biology</b> [BIO190, QMUL] <i>(2019/20).</i>
* <b>Practical laboratories and Seminars</b> [SEF033, QMUL] <i>(2018/19)</i>.
* <b>Stats & R</b> [BIO309, QMUL] <i>(2017/18 – 2019/20).</i>
* <b>Free style laboratories</b> [BIO193 & BIO199, QMUL] <i>(2016/17).</i>
* <b>Thinking workshops</b> [BIO193 & BIO199, QMUL] <i>(2016/17 – 2017/18).</i>
* <b>Dry lab tutorials</b> [BIO193 & BIO199, QMUL] <i>(2016/17).</i>
* <b>Biochemistry laboratories</b> [BIO199, QMUL] <i>(2016/17).</i>

<br>

## __Invited Talks and Science Communication <i>(outreach)</i>__

* <b>The importance of data accessibility and reproducibility in computational research</b> <i>(Jun 2025) – Invited talk at the UGI meetings at University College London.</i>
* <b>Application of Bayesian model selection to distinguish between species diversification timing hypotheses</b> <i>(May 2025) – Invited talk at the "Ancient Rocks and Clocks meeting" that took place at McGill University in Montréal.</i>
* <b>Fossil calibrations in Bayesian relaxed-clock dating</b> <i>(Mar 2025) – Invited talk at the "Symposium on Computational Phylogenomics" that took place at Southern University of Science and Technology (SUSTech) in Shenzhen.</i>
* <b>Bayesian methods to infer evolutionary timelines when deep divergences are present and large genomic datasets used</b> <i>(Jul 2024) – Invited talk at the [Genome Science conference](https://bristol.genomescience.org.uk/committee/) that took place at the University of Bristol.</i>
* <b>Bayesian phylogenomic dating: a journey on reproducible timetree inference</b> <i>(Jun 2024) – Invited talk at the [“11<sup>th</sup> International Biennial Conference" organised by the Systematics Association](https://systass.org/events/biennial/) that took place at the University of Reading.</i>
* <b>XXIII International Congress of Genetics</b> <i>(Jul 2023) – Invited talk at the “Molecular Evolution” Symposium at the Evolutionary Genetics session.</i>
* <b>Native Schools workshop</b> <i>(Nov 2022) – Scientific activity sponsored by Native Scientist aimed at promoting science and language among school pupils.</i>
* <b>Silwood Ecology and Evolution Seminar, Imperial College London</b> <i>(Jun 2022) – Invited talk (virtual).</i>
* <b>“Data, tools and applications: Creating a pipeline from Omics to AI” meeting, Alan Turing Institute</b> <i>(May 2022) – Invited talk.</i>
* <b>Brunel Bioinformatics Workshop 2022</b> <i>(May 2022) – Invited talk at the [“Statistical Modelling” session of the workshop](https://bbw2022.co.uk/speakers).</i>
* <b>Interview at “En Fase Experimental” radio podcast</b> <i>(May 2022) – Interview about the day-to-day of a computational biologist and my research in evolutionary biology (virtual). [You can listen to this podcast episode on the SRUK/CERU website](https://sruk.org.uk/es/efex64/)</i>
* <b>Seminar on Phylogenomics analyses with `MCMCtree`, University of Oklahoma</b> <i>(Apr 2022) – Invited talk (virtual).</i>
* <b>Interview by the students at IE Cañada Blanch for the International Day of Women and Girls in Science</b> <i>(Feb 2022) – Interview about my research and how it feels to pursue a career as a woman in STEM. You can see the videos with my answers to the students' questions [on YouTube](https://www.youtube.com/playlist?list=PLy_Y_fVXS6UxEmAPatrdVrLYzxAzDNNtP). This event was a collaboration between [IE Cañada Blanch](https://twitter.com/Canada_Blanch) and the [Office for Cultural and Scientific Affairs (Embassy of Spain in London)](https://twitter.com/SpainCultureUK).</i>
* <b>Interview at “Evolution Soup” YouTube channel</b> <i>(Jan 2022) – Interview about my research on mammals’ evolution and my academic career [available to watch on YouTube](https://www.youtube.com/watch?v=shsOpkkcq_A&feature=youtu.be) (virtual).</i>
* <b>Seminar at the Instituto de Ciencias Matemáticas, Consejo Superior de Investigaciones Científicas</b> <i>(Nov 2021) – Invited talk (virtual).</i>
* <b>Seminar at the Palaeobiology Research group, University of Bristol</b> <i>(Jun 2021) – Invited talk (virtual).</i>
* <b>Graduate Phylogenomics seminar, Michigan State University</b> <i>(Feb 2021) – Invited talk (virtual).</i>
* <b>CLOE Joint Group Meeting, University College London</b> <i>(Nov 2020) – Invited talk (virtual).</i>
* <b>Back to present: Understanding evolution</b> <i>(Jun 2018 & Jun 2019) – Scientific activity for children presented during the V and VI SRUK/CERU International Symposia.</i>

<br>

## __Events Organisation and Participation__

* <b>Bayesian Phylogenetics Workshop</b> <i>(Jan 2025 – Apr 2025) – [Co-organiser of the workshop](https://systass.org/workshop-bayesian-phylogenetics/).</i>
* <b>12<sup>th</sup> SRUK/CERU International Symposium</b> <i>(Nov 2024 – Jul 2025) – [Website manager, developing and maintaining the website](https://symposium.sruk.org.uk/event-2025/).</i>
* <b>11<sup>th</sup> SRUK/CERU International Symposium</b> <i>(Nov 2023 – Jul 2024) – [Website manager, developing and maintaining the website](https://symposium.sruk.org.uk/event-2024/).</i>
* <b>XXIII International Congress of Genetics</b> <i>(Nov 2022 – Jul 2023) – Co-convenor at the “Molecular Evolution” symposium at the Evolutionary Genetics session.</i>
* <b>X SRUK/CERU International Symposium</b> <i>(Nov 2022 – Jul 2023) – Member of the organising committee ([website development and management](https://sruk2023.wordpress.com/)).</i>
* <b>CLOE Winter Quarterly Symposium</b> <i>(Oct – Dec 2021) – Member of the organising committee of the CLOE anniversary symposium. You can find [more information about this event and the invited speakers at the CLOE website](https://www.ucl.ac.uk/biosciences/gee/ucl-centre-lifes-origins-and-evolution)</i>
* <b>IX SRUK/CERU International Symposium</b> <i>(Nov 2021 – Jun 2022) – Member of the organising committee ([website development and management](https://sruk2022.wordpress.com/)).</i>
* <b>SSB Awards Committee</b> <i>(Jun 2021) – Member of the judging panel for the talks presented at the SSB Excellence Symposium.</i>
* <b>VIII SRUK/CERU International Symposium (virtual)</b> <i>(Nov 2020 – Jun 2021) – Member of the organising committee ([website development and management](https://sruk2021.wordpress.com/)).</i>
* <b>VII SRUK/CERU International Symposium</b> <i>(Nov 2018 – Jun 2019) – Member of the organising committee ([website development and management](https://sruk2019.wordpress.com/)).</i>
* <b>VI SRUK/CERU International Symposium</b> <i>(Mar – May 2018) – Member of the organising committee ([website development and management](https://sruk2018symposium.wordpress.com/)).</i>
* <b>Phylogroup XI</b> <i>(Dec 2017 – Apr 2018) – Co-organiser of the meeting.</i>

<br>

## __Research Presentations__

* <b>Application of Bayesian model selection to distinguish between species diversification timing hypotheses</b> <i>(Aug 2024) – In-person talk at the "Phylogenetics" session that was part of the MASAMB meeting in London at Queen Mary University of London.</i>
* <b>Using a Bayesian cross-bracing approach to infer the age of the Last Universal Common Ancestor</b> <i>(Jul 2024) – In-person talk at the "Computational Biology II" session that was part of the Evolution meeting in Montréal.</i>
* <b>Inferring species level timelines with phylogenomic data: A Bayesian sequential subtree approach</b> <i>(Jun and Aug 2022) – Virtual talk at the online session of the Evolution meeting and in-person talk at the ESEB conference.</i>
* <b>Estimating the divergence of dire wolves from their common ancestor with living canids using a Bayesian dating approach</b> <i>(Jun 2021) – Talk at the online Evolution meeting.</i>
* <b>Estimating the divergence of dire wolves from their common ancestor with living canids using a Bayesian dating approach</b> <i>(Nov 2019) – Poster presentation at the Young Systematists’ Forum meeting.</i>
* <b>Bayesian estimation of species divergence times using correlated quantitative characters</b> <i>(Jun 2019) – [Talk at the Evolution meeting awarded with the Ernst Mayr Award](https://www.systbio.org/ernst-mayr-award.html).</i>
* <b>Bayesian estimation of species divergence times using correlated quantitative characters</b> <i>(Nov 2018) – Talk at the Young Systematists’ Forum meeting.</i>
* <b>Comparison of relaxed-clock models using the Bayesian model selection implemented in `MCMCtree`</b> <i>(Aug 2018) – Poster presentation at the Evolution meeting, available at the [dos Reis Lab website](https://dosreislab.github.io/2018/08/22/evolution2018.html).</i>
* <b>Bayesian estimation of species divergence times using quantitative characters</b> <i>(Apr 2018) – Talk at Phylogroup XI meeting.</i>

<br>

## __Society Memberships__

* <b>The Systematics Association</b> <i>(Jul 2024 – Present)</i> – Regular member of the [Systematics Association](https://systass.org/).
* <b>SMBE</b> <i>(Sep 2022 – Present)</i> – Regular member of the [Society for Molecular Biology and Evolution](https://www.smbe.org/smbe/).
* <b>ESEB</b> <i>(May 2022 – Present)</i> – Regular member of the [European Society of Evolutionary Biology](https://eseb.org/).
* <b>SSE</b> <i>(Feb 2019 – Present)</i> – Regular member of the [Society for the Study of Evolution](http://www.evolutionsociety.org/).
* <b>SSB</b> <i>(Jan 2018 – Present)</i> – Regular member of the [Society of Systematic Biologists](https://www.systbio.org/). Involvement in the Society:
  * 2021-2024: Reviewer for the [Graduate Student Research Awards](https://www.systbio.org/graduate-student-research-awards.html).
  * In 2021 and 2024: Member of the judging panel for the [SSB Ernst Mayr Award](https://www.systbio.org/ernst-mayr-award.html).
  * Since 2023: Mentor at the [SSB Mentoring Program](https://www.systbio.org/mentorship-program.html).
* <b>SRUK/CERU</b> <i>(Nov 2017 - Present)</i> – Regular member of the [Society of Spanish Researchers in the United Kingdom (SRUK/CERU)](https://sruk.org.uk). Involvement in the Society:
  * Active roles:
    * Since 2024: Co-chair of the [Evolution Committee](https://sruk.org.uk/teams/evolution-committee/) (Founding Member).
    * 2017 & 2025: Mentor at the [SRUK/CERU Mentoring Programme](https://sruk.org.uk/initiatives/mentoring/).
  * Past roles ([see my SRUK/CERU Profile](https://sruk.org.uk/team/sandra-ac/)):
    * Jul 2024 - Jul 2025: [Treasurer (Board of Trustees)](https://find-and-update.company-information.service.gov.uk/officers/YjMWtjepxtiW8j8puYJN2YSMXZo/appointments).
    * Sep 2020 - Jul 2025: Website manager. Websites maintained: [main SRUK/CERU's website](https://sruk.org.uk/), [Symposium's website](https://symposium.sruk.org.uk/), [Members Zone](https://members.sruk.org.uk/).
    * Oct 2017- Apr 2025: [Member of the SRUK/CERU London working group](https://sruk.org.uk/constituencies/london/).
    * Oct 2019- Jul 2025: [Co-editor-in-chief of the SRUK/CERU blog](https://sruk.org.uk/blog/).
    * Jul 2019 - Jul 2021: Secretary of the [SRUK/CERU London constituency](https://sruk.org.uk/constituencies/london/).
    * Jul 2021 - Jul 2023: Director of the former SRUK/CERU Press Department, now replaced by the [SRUK/CERU Outreach and Public Engagement Department](https://sruk.org.uk/teams/outreach-department/). Member of the former Press Department since 2018.

<br>

## __Peer-review__

Manuscripts reviewed for <i>Systematic Biology</i>, <i>Bioinformatics</i>, <i>Methods in Ecology and Evolution</i>, <i>Evolution</i>, <i>Cambridge Elements STM3</i>, <i>Nature Ecology and Evolution</i>, <i>Paleobiology</i>, <i>Proceedings of the Royal Society B</i>, <i>Nature Communications</i>, <i>Communications Biology</i>, <i>Bioinformatics Advances</i>, <i>Nature</i>, <i>Palaeontology</i>, <i>Genome Biology and Evolution</i>, <i>Molecular Biology and Evolution</i>.

<space></space>

<br>

## __Additional Information__

Native speaker of Spanish, fluent in English and Catalan, basic level in Greek and Modern Standard Arabic. Mysore Ashtanga practitioner. Piano player and amateur composer.

Since 2022, I administer the [PAML discussion group](https://groups.google.com/g/pamlsoftware?pli=1) and handle most of users requests. I have also developed the [interactive `PAML` Wiki](https://github.com/abacus-gene/paml/wiki) and [various tutorials](https://github.com/abacus-gene/paml-tutorial) with which I hope `PAML` users can make the most out of the widely used programs within the `PAML` software.

<br>

---
<spacer></spacer>

Last update: 2025/07/23.


