---
layout: page
title: About
permalink: /about/
---

![image](/assets/figs/sac.jpg){: style="float: right" width="20%"}
Hi everyone!

I am Sandra Álvarez-Carretero, a computational scientist working as a post-doctoral researcher and teaching fellow at [Ziheng Yang Lab](http://abacus.gene.ucl.ac.uk/) at University College London.



Even though now I cannot imagine a day working without my laptop, my career started
at the wet lab and, little by little, I specialised in computer science:

<br />
<br />

* __BSc in Biochemistry and Molecular Biology__ *([Universidad Rovira I Virgili](http://www.urv.cat/es/estudios/grados/oferta/graudebioquimica/), Spain)*
: During this 4-year BSc, most of my practical sessions involved wet-lab experiments. Nevertheless, it was not until I took the bioinformatics course part of my BSc that I realised how much 
I enjoyed coding as I could tell my computer to do __"something"__ with my 
biological data and I actually got __"this something"__ done 
(what I could not always say after my wet laboratory practical sessions...). Thanks to the Erasmus+ programme, I could work on a bioinformatics project at the [University of Manchester](http://www.manchester.ac.uk/),
which was focused on troubleshooting and implementing a pipeline to carry out virtual screenings.
From then onwards, I was sure that my next steps in science 
had to integrate two fields: biology and informatics.  

* __MSc in Bioinformatics__ *([Högskolan I Skövde](http://www.his.se/en/#), Sweden)*
: After finishing the theoretical courses, I carried out my Master's dissertation at [Karolinska Institutet](http://ki.se/en/research/centre-for-translational-microbiome-research-ctmr), 
where I developed a pipeline to characterise bacterial isolates based on whole-genome sequencing data
 and other software pipelines to analyse transcriptomic and proteomic data. If you are interested in reading my dissertation, 
 "BACTpipe: Characterization of bacterial isolates based on whole-genome sequence data", you can download it from [here](http://his.diva-portal.org/smash/record.jsf?pid=diva2%3A1197064&dswid=-2253). `BACTpipe` has now been further implemented by the computational research team at the [CTMR](https://ki.se/en/research/centre-for-translational-microbiome-research-ctmr). You can find the open source code in [this GitHub repository](https://github.com/ctmrbio/BACTpipe).

* __PhD student__ *([Queen Mary University of London](http://www.sbcs.qmul.ac.uk/), UK)*
: Once I finished my MSc, I decided to explore other fields in computer science such as 
Bayesian statistics and modelling so I could learn more about how I could merge my computing knowledge with the maths used in these two fields. 
I carried out my PhD at [dos Reis Lab](https://dosreislab.github.io/people/) research group, at Queen Mary University of London,
where I worked on the application of Bayesian MCMC statistical methods to 
study species divergences through time with both molecular and morphological quantitative data.

* __Post-doctoral researcher and Teaching fellow__ *([University College London](https://www.ucl.ac.uk/biosciences/departments/genetics-evolution-and-environment), UK)*
: Currently, my main interest is focused on developing and applying new methods to estimate species divergence times with molecular and morphological
quantitative datasets. I aim to use these tools in Bayesian dating analyses when combining phylogenomic datasets with large-scale 
morphological datasets (with both extant and fossil species). My ultimate goal is to ensure that all the pipelines and tools that
I develop as part of the projects I am involved in are well documented, so they can be easily used to reproduce any reported results generated with them.

<br />


# __Publications__

* [A Species-Level Timeline of Mammal Evolution Integrating Phylogenomic Data](https://rdcu.be/cDHW7)   
   __Álvarez-Carretero S__<sup>*</sup>, Tamuri AU<sup>*</sup>, et al. (2021) Nature.<br><font size="2"><sup>*</sup>These authors contributed equally.</font><br />   
   > The step-by-step tutorial that you can follow to reproduce the results of this study can be found [here](https://github.com/sabifo4/mammals_dating).   
   > In addition, you can watch [here](https://www.youtube.com/watch?v=shsOpkkcq_A&feature=youtu.be) the interview I had at Evolution soup with regards to this research   
   > on the most precise evolutionary timeline for mammals to learn more about each author's contribution, the outline and main outcome of this research, and a little bit more about my academic career.   
   > The press release we wrote about this study can be found both in [English](https://sruk.org.uk/study-offers-new-insights-into-the-timeline-of-mammal-evolution/) and [Spanish](https://sruk.org.uk/es/un-nuevo-estudio-proporciona-nuevas-respuestas-sobre-la-cronologia-evolutiva-de-los-mamiferos/).   
   
* [Bayesian phylogenomic dating](https://link.springer.com/chapter/10.1007/978-3-030-60181-2_13)   
   __Álvarez-Carretero S__ and dos Reis M. (2020) _In_: Ho S (ed.) The Molecular Evolutionary Clock: Theory and Practice (2021).

* [Dire wolves were the last of an ancient New World canid lineage](https://www.nature.com/articles/s41586-020-03082-x)   
   Perri Angela R<sup>*</sup>, Mitchell Kieren J<sup>*</sup>, Mouton Alice<sup>*</sup>, __Álvarez-Carretero S__<sup>*</sup>, et al. (2021) Nature.<br><font size="2"><sup>*</sup>These authors contributed equally.</font><br />

* [Phylogenomic Resolution of the Cetacean Tree of Life Using Target Sequence Capture](https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz068/5601630)   
   McGowen MR, Tsagkogeorga G, __Álvarez-Carretero S__, dos Reis M, Struebig M, Deaville R, Jepson PD, Jarman S, Polanowski A, Morin PA, Rossiter SJ. (2019) Systematic Biology.
   
* [Identification of Functional and Druggable Sites in Aspergillus fumigatus Essential Phosphatases by Virtual Screening](https://www.mdpi.com/1422-0067/20/18/4636/htm)   
   Thornton BP, Johns A, Al-Shidhani R, __Álvarez-Carretero S__, Storer ISR, Bromley MJ, Tabernero L. (2019) Molecules.   

* [Bayesian Estimation of Species Divergence Times Using Correlated Quantitative Characters](https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz015/5366706)  
   __Álvarez-Carretero S__, Goswami A, Yang Z, dos Reis M. (2019) Systematic Biology.   
   > The step-by-step tutorial followed to carry out each computational task of this study can be found [here](https://github.com/sabifo4/morpho).   
   > The R functions described in this study were implemented in the `mcmc3r` R package, available for download [here](https://github.com/dosreislab/mcmc3r). 

* [The bile salt glycocholate induces global changes in gene and protein expression and activates virulence in enterotoxigenic Escherichia coli](http://www.mdpi.com/1420-3049/23/2/353)  
   Joffre E, Nicklasson M, __Álvarez-Carretero S__, Xiao X, Sun L, Nookaew I, Zhu B, Sjöling Å. (2019) Scientific Reports.

* [VSpipe, an Integrated Resource for Virtual Screening and Hit Selection: Applications to Protein Tyrosine Phospahatase Inhibition](http://www.mdpi.com/1420-3049/23/2/353)  
   __Álvarez-Carretero S__, Pavlopoulou N, Adams J, Gilsenan J, and Tabernero L. (2018) Molecules.   
   > `VSpipe` is an open-source code available for download [here](https://github.com/sabifo4/VSpipe).

* [An evaluation of different partitioning strategies for Bayesian estimation of species divergence times](https://academic.oup.com/sysbio/article/doi/10.1093/sysbio/syx061/3921199/An-Evaluation-of-Different-Partitioning-Strategies)  
   Konstantinos A, __Álvarez-Carretero S__, dos Reis M, and Yang Z. (2017) Systematic Biology.

<br />

# __Outreach, events organisation, and research presentations__

#### *Academic course 2017-2018* 

   * **NERC DTP induction at QMUL** *(Nov 2017)*  
      Overview of the work carried out at dos Reis lab and about my PhD project. The practical part was focused on understanding the usage of the bioinformatics tools we use followed by a quick guide on a morphometrics analysis (acess [here](https://sabifo4.github.io/blog/Morphometrics_and_Procrustes_alignment)). 

   * **Phylogroup XI** *(Dec 2017 - Apr 2018)*  
      During these four months, I supported the organisation of the Phylogroup XI event. My main tasks were to collect the proposals for the oral presentations, to help decide which talks would be selected for the event, and organise the schedule of the event. 

   * **VI International Symposium SRUK/CERU** *(Mar 2018 - May 2018)* 
      During almost three months, I maintained and developed the website of the event as well as of the symposium forum for accommodation in Glasgow. 

   * **Bayesian estimation of species divergence times using quantitative characters** *(Apr 2018)*  
      Talk at Phylogroup XI about the preliminary results of my PhD research.
      
   * **Scientific Computing Workshop: A practical introduction to R and Python for researchers** *(Apr 2018)*   
      Event sponsored by the SRUK/CERU society that I co-organized with Dr. Diego Alonso.        

   * **Introduction to R** *(May 2018 | Computational Molecular Evolution course)*  
      Two-morning sessions about programming in R that I organized during the Computational Molecular Evolution course in Crete.  

   * **Back to present: Understanding evolution** *(Jun 2018)*  
      Activity that I prepared for the Science Zone as part of the VI International Symposium SRUK/CERU in Glasgow.  

   * **Comparison of relaxed-clock models using the Bayesian model selection implemented in MCMCtree (_Evolution 2018 meeting, poster_)** *(Aug 2018 | Evolution 2018 meeting, poster)*  
      Poster presentation at the joint Evolution 2018 meeting showing how to use the mcmc3r R package to estimate the marginal likelihood using the likelihood collected during MCMC chains ran in MCMCtree and compute Bayes factors for model selection. You can find the poster [here](https://dosreislab.github.io/2018/08/22/evolution2018.html).
      
#### *Academic course 2018/2019* 

   * **NERC DTP induction at QMUL** *(Nov 2018)*  
      Overview of the work carried out at dos Reis lab and a hands-on tutorial on how to generate a molecular alignment, estimate a molecular tree, and process morphometrics data.
      
   * **Bayesian estimation of species divergence times using correlated quantitative characters** *(Nov 2018 | YSF18 meeting, talk)*  
      Talk at the YSF2018 1-day symposium on the paper already submitted about the main method I had been working on my PhD.  
      
   * **VII International Symposium SRUK/CERU** *(Nov 2018 - Jun 2019)* 
      I was part of the organising committee of the symposium. My main task during these months was to develop and maintain the website to keep up with the latest updates regarding the event.   

   * **II Scientific Computing Workshop: A practical introduction to R and Python for researchers** *(Mar 2019)*   
      Event sponsored by the SRUK/CERU society that I co-organized with Dr. Diego Alonso.   
     
   * **Introduction to R** *(May 2019 | Computational Molecular Evolution course)*  
      One-morning session about programming in R that I organized during the Computational Molecular Evolution course in Hinxton.
      
   * **Back to present: Understanding evolution** *(Jun 2018)*  
      Activity that I prepared for the Science Zone as part of the VII International Symposium SRUK/CERU in Liverpool. 

   * **Bayesian estimation of species divergence times using correlated quantitative characters** *(June 2019 | Evolution 2019 meeting, awards talk)*  
      Talk at the Evolution 2019 meeting, with which I was granted the [**Ernst Mayr award**](https://www.systbio.org/ernst-mayr-award.html).
      
#### *Academic course 2019/2020* 

   * **NERC DTP induction at QMUL** *(Nov 2019)*  
      Overview of the work carried out at dos Reis lab and a hands-on tutorial on how to generate a molecular alignment, estimate a molecular tree, and process morphometrics data.   
      
   * **Estimating the divergence of dire wolves from their common ancestor with living canids using a Bayesian dating approach** *(Nov 2019 | YSF19 meeting, poster)*   
      Poster presentation at the YSF2019 1-day symposium on the submitted manuscript about the evolution of canids with a special focus on the dispersal and speciation of dire wolves.   
 
   * **SRUK Research Computing Workshop - R, Bash and Python** *(Feb & Jun 2020, two sessions)*   
      Events sponsored by the SRUK/CERU society that I co-organized with Dr. Diego Alonso.   

#### *Academic course 2020/2021* 
      
   * **NERC DTP induction at QMUL** *(Nov 2020)*  
      Overview of the work carried out at dos Reis lab and a hands-on tutorial on how to generate a molecular alignment, estimate a molecular tree, and process morphometrics data.   
      
   * **Bayesian estimation of species divergence times using GMM data** *(Nov 2020)*   
      Talk at the CLOE Joint Group Meeting organised by UCL.   
      
   * **VIII International Symposium SRUK/CERU** *(Nov 2020 - Jun 2021)* 
      I was part of the organising committee of the symposium. My main task during these months was to develop and maintain the website to keep up with the latest updates regarding the event.   

   * **Estimating the divergence of dire wolves from their common ancestor with living canids using a Bayesian dating approach** *(Jun 2021 | Evolution 2021 online conference, talk)*   
      Talk at the Virtual Evolution 2021 conference.   
      
   * **A Bayesian approach to estimate species divergence times using geometric morphometrics data** *(Jun 2021)*   
      Talk as part of the seminars organised by the Palaebiology Research group at University of Bristol.   
      
#### *Academic course 2021/2022* 

   * **NERC DTP induction at QMUL** *(Nov 2021)*  
      Overview of the work carried out at dos Reis lab and a hands-on tutorial on how to generate a molecular alignment, estimate a molecular tree, and process morphometrics data.   
      
   * **Breaking down the steps in Bayesian phylogenomic dating** *(Nov 2021)*   
      Talk as part of the seminars organised by Instituto de Ciencias Matmáticas at CSIC.   

   * **IX International Symposium SRUK/CERU** *(Nov 2021 - Jul 2022)* 
      I was part of the organising committee of the symposium. My main task during these months was to develop and maintain the website to keep up with the latest updates regarding the event.   
      
   * **Phylogenomics course** *(Dec 2021)*   
      Workshop organised by Physalia where I taught the practical sessions of Bayesian inference analyses. See resources [here](https://github.com/sabifo4/BayesianInference_practicals).   
      
   * **CLOE Winter Quarterly Symposium** *(Dec 2021)*   
      Part of the organising committee of the CLOE anniversary symposium.   
   
   * **Interview at Evolution Soup** *(Jan 2022)*   
      Interview I had at Evolution soup with regards to my research contribution to [the study on the most precise evolutionary timeline for mammals](https://rdcu.be/cDHW7). You will learn a little bit more about my academic career, the outline of this research study and how each author contributed to this project, how we can apply Bayesian stats to evolutionary biology studies, and what it feels like working as a computational biologist/bioinformatician in academic research.   
      You can watch the interview [here](https://www.youtube.com/watch?v=shsOpkkcq_A&feature=youtu.be).   
      
   * **International Day of Women and Girls in Science** *(Jan/Feb 2022)*   
      Interview with the students at IE Cañada Blanch, where I talked about my career as a computational biologist and answered the students' questions about how I feel as a woman pursuing a career in science. You can see the videos with my answers to the students' questions [here](https://www.youtube.com/playlist?list=PLy_Y_fVXS6UxEmAPatrdVrLYzxAzDNNtP).   
      This was a collaboration between [IE Cañada Blanch](https://twitter.com/Canada_Blanch) and the [Office for Cultural and Scientific Affairs (Embassy of Spain in London)](https://twitter.com/SpainCultureUK).   
      
   * **Interview at “En Fase Experimental”** *(May 2022)*   
      Interview as part of a radio podcast in which I talked about the day-to-day of a computational biologist and my research in evolutionary biology as a woman in
      STEM. You can listen to the podcast [here](https://sruk.org.uk/es/efex64/).   
      
   * **_Statistical Modelling_ session at Brunel University** *(May 2022)*   
      I was an invited speaker at the Brunel Bioinformatics Workshop 2022, where I presented my research in mammal evolution and discussed bioinformatics analyses with
      a short tutorial. More information about the workshop [here](https://bbw2022.co.uk/speakers).   
      
   * **Data, tools and applications: Creating a pipeline from Omics to AI** *(May 2022)*   
      I was an invited speaker at the the Alan Turing Institute to discuss my research in mammal evolution as well as the challenges to use AI when using phylogenomics
      datasets.   
      
   * **Silwood Ecology and Evolution Seminar** *(Jun 2022)*   
      I was an invited speaker at the the Silwood Ecology and Evolution Seminar, where I talked about how to carry out Bayesian dating analyses with a short demo and
      included an example with my research in mammal evolution.   
      
   * **Inferring species level timelines with phylogenomic data: A Bayesian sequential subtree approach** *(Jun and Aug 2022 | Evolution 2022 and ESEB 2022, talks)*   
      Virtual talk at the Evolution 2021 conference and in-person talk at the ESEB conference.   
      
<br />

# __Teaching__

#### *Academic course 2016 - 2017*  
   
   * **Demonstrator (TA): Biochemistry laboratories**  
   Module BIO199, Undergraduate level (year 1), Queen Mary University of London.
   
   * **Demonstrator (TA): Dry lab tutorials**  
   Modules BIO193 and BIO199, Undergraduate level (year 1), Queen Mary University of London.
   
   * **Demonstrator (TA): Thinking workshops**  
   Modules BIO193 and BIO199, Undergraduate level (year 1), Queen Mary University of London.
   
   * **Demonstrator (TA): Free style laboratories**  
   Modules BIO193 and BIO199, Undergraduate level (year 1), Queen Mary University of London.

#### *Academic course 2017 - 2018*
   
   * **Demonstrator (TA): Stats & R**  
   Module BIO309, Undergraduate level (year 3), Queen Mary University of London.
   
   * **Demonstrator (TA): Thinking workshops**  
   Modules BIO193 and BIO199, Undergraduate level (year 1), Queen Mary University of London.   
   
   * **Teaching Assistant (TA): Computational Molecular Evolution course**   
   Providing support to the participants which ranged from software installation and usage to the theoretical background of using these bioinformatics tools to study molecular evolution.
     
#### *Academic course 2018 - 2019*
   
   * **Demonstrator (TA): Stats & R**   
   Module BIO309, Undergraduate level (year 3), Queen Mary University of London.
   
   * **Demonstrator (TA): Practical laboratories and Seminars**   
   Modules SEF033, Foundation level, Queen Mary University of London.   
   
   * **Teaching Assistant (TA): Computational Molecular Evolution course**   
   Providing support to the participants which ranged from software installation and usage to the theoretical background of using these bioinformatics tools to study molecular evolution.
 
#### *Academic course 2019 - 2020*
   
   * **Demonstrator (TA): Stats & R**  
   Module BIO309, Undergraduate level (year 3), Queen Mary University of London.
   
   * **Demonstrator (TA): IT stats for Molecular and Cellular Biology practicals**  
   Module BIO190, Undergraduate level (year 1), Queen Mary University of London.
   
   * **Demonstrator (TA): Statistics for Bioinformaticians**  
   Module BIO782P, Graduate level (MSc programme in Bioinformatics), Queen Mary University of London.   
   
#### *Academic course 2020 - 2021*
   
   * **Instructor: Molecular Evolution**  
   Module BIOL0033, MSci programme (year 4), University College of London.
    
#### *Academic course 2021 - 2022*
   
   * **Teaching fellow: Biochemistry and Molecular Biology**  
   Module BIOC0001, BSc programme (year 1), University College of London.   
   
   * **Teaching fellow: Basics for Molecular Biosciences**  
   Module BIOC0002, BSc programme (year 1), University College of London.   
   
   * **Teaching fellow: Introdution to Microbiology**  
   Module BIOC0004, BSc programme (year 1), University College of London.  
   
   * **Teaching fellow: Biochemistry and Molecular Biology**  
   Module BIOC0005, BSc programme (year 1), University College of London.  
   
   * **Teaching fellow: Advanced Investigative projects in molecular biosciences**  
   Module BIOC0021, BSc programme (year 3), University College of London.   
 
   * **Teaching fellow: MSci Biochemistry Year 4**  
   Modules BIOC0024/BIOC0025/BIOC0026, MSci programme (year 4), University College of London.   
   
   * **Instructor: Molecular Evolution**   
   Module BIOL0033, MSci programme (year 4), University College of London.   
   
   * **Instructor: Bayesian statistics**   
   Masters in Computational Methods in Ecology and Evolution, Imperial College London.   
   
   * **Teaching Assistant (TA): Computational Molecular Evolution course**   
   Providing support to the participants which ranged from software installation and usage to the theoretical background of using these bioinformatics tools to study
   molecular evolution.
   
<br />

# __Awards and scholarships__

   * [**Scholarship for research funding during the academic course 2014-15**](https://www.boe.es/boe/dias/2015/01/27/pdfs/BOE-A-2015-700.pdf), awarded by Ministerio de Educación, Cultura y Deporte in 2014.   
   (Original award name: _Becas de Colaboración 2014-15_)    
   * [**Student excellence award for the BSc in Biochemistry and Molecular Biology**](https://seuelectronica.urv.cat/media/upload/pdf//cg_20160225_punt71_premis_ext_grau_mast_corrsl_pend_sig_editora_4_1_2128.pdf), awarded by Universitat Rovira I Virgili.   
   (Original award name: _Premi Extraordinari de Final d’Estudis del grau de Bioquímica I Biologia Molecular_)   
   * [**National awards after University degree completion during the 2014-15 academic course**](http://www.educacionyfp.gob.es/servicios-al-ciudadano/catalogo/general/99/998341/ficha/998341-2014-2015.html), awarded by Ministerio de Educación, Cultura y Deporte.   
   (Original award name: _Premios Nacionales de Fin de Carrera de Educación Universitaria para el curso académico 2014-2015_)   
   * [**Ernst Mayr Award**](https://www.systbio.org/ernst-mayr-award.html), awarded by the Society of Systematic Biologists.

<br />

# __Society Memberships__

   * **SRUK/CERU** - Society of Spanish Researchers in the UK  *(Nov 2017 - Present)*   
      Member of the SRUK/CERU London working group since 2017 and former secretary of the [SRUK/CERU London constituency](https://sruk.org.uk/constituencies/london/) (2019-2021). Member of the Press Department since 2018, [**co-editor-in-chief of the SRUK/CERU blog**](https://sruk.org.uk/teams/blog-committee/) since 2019, and current [**director of the Press department**](https://sruk.org.uk/teams/press-department-backstage/). SRUK/CERU Website manager since 2021.   
   * **SSB** – Society of Systematic Biologists *(Jan 2018 - Present)*
   * **SSE** – Society for the Study of Evolution *(Feb 2019 - Present)* 

<br />

# __Peer-review__

Reviewer for _Systematic Biology_, _Bioinformatics_, _Methods in Ecology and Evolution_, _Evolution_, _Cambridge Elements STM3_, _Nature Ecology and Evolution_, _Paleobiology_, _Proceedings of the Royal Society B_, _Nature Communications_, _Communications Biology_, and _Bioinformatics Advances_, _Nature_, _Genome Biology and Evolution_.

----

Last update: 2022/10/13.
